lib/config/options/index.ts

```diff
@@ -1766,7 +1766,7 @@ const options: RenovateOptions[] = [
   {
     name: 'keepUpdatedLabel',
     description:
-      'Label to request Renovate bot always rebase to keep branch updated.',
+      'If set, users can add this label to PRs to request they be kept updated with the base branch.',
```

Comment:
Does this change make this a `fix` type PR? Or can I keep using the `docs` scope for it?

Reply:
docs scope